cmd/tlin/main.go

```diff
@@ -92,6 +93,7 @@ func parseFlags(args []string) Config {
 	flagSet.BoolVar(&config.CFGAnalysis, "cfg", false, "Run control flow graph analysis")
 	flagSet.StringVar(&config.FuncName, "func", "", "Function name for CFG analysis")
 	flagSet.BoolVar(&config.AutoFix, "fix", false, "Automatically fix issues")
+	flagSet.StringVar(&config.Output, "o", "", "Output path of GraphViz file in a SVG format")
```

Comment:
I just had a thought: what about standarize the file output flag to `-o`?

If we applying this, we could handle JSON output in the same way with `-o`, which might reduce cognitive load for users.

Reply:
mmh, it could be a good idea to make a general output path and change json-output to just json to specify to use a json